dipy/reconst/tests/test_qtdmri.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+from dipy.data import get_gtab_taiwan_dsi
+from numpy.testing import (assert_almost_equal,
+                           assert_array_almost_equal,
+                           assert_equal,
+                           run_module_suite)
+from dipy.reconst.shore_time import ShoreTemporalModel
+from dipy.reconst import maptime
+from dipy.sims.voxel import (MultiTensor, all_tensor_evecs,  multi_tensor_pdf)
+from scipy.special import gamma
+from scipy.misc import factorial
+from dipy.data import get_sphere
+from dipy.sims.voxel import add_noise
+import scipy.integrate as integrate
+import scipy.special as special
```

Comment:
This import are not used : 

 from scipy.special import gamma
 from scipy.misc import factorial
 from dipy.data import get_sphere
 from dipy.sims.voxel import add_noise
 import scipy.integrate as integrate
 import scipy.special as special